### PR TITLE
Perf

### PR DESCRIPTION
**Pull request by denar90**

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -70,40 +68,32 @@ <h3 class="f4 br3 br--top black-60 pv2 mb0">Wanna compare traces?</h3>
...
+    <script src="storage.js" defer></script>
+    <script src="timeline_viewer.js" defer></script>
     <script>
-      function init() {
+      if ('serviceWorker' in navigator) navigator.serviceWorker.register('service-worker.js');
+
+      document.addEventListener('DOMContentLoaded', () => {
         // only works in Chrome because browser devtools
-        if (!(window.chrome && chrome.loadTimes)) {
+        if (!window.chrome) {
```
Comment (denar90) [inline docs/index.html]: due to https://bugs.chromium.org/p/chromium/issues/detail?id=621512

**Comment by denar90** [comment]
@paulirish I 😍  this staging stuff 👍  Will be also nice to extend apiKey access to staging pages.

**Comment by paulirish** [comment]
wfm!

**Comment by paulirish** [comment]
> Will be also nice to extend apiKey access to staging pages.

added this to the API key
image

i cant add these domains to the oath client ID because it says:

> For use with requests from a browser. This is the origin URI of the client application. It can't contain a wildcard (http://*.example.com) or a path (http://example.com/subdir). If you're using a nonstandard port, you must include it in the origin URI.

but maybe just API key is fine? i can't tell. :)

**Pull request merged**